package/network/services/openvpn/files/openvpn.init

```diff
@@ -73,12 +73,30 @@ openvpn_add_instance() {
 	local dir="$2"
 	local conf="$3"
 
+	local up down
+
+	config_get up "$name" up
+	[ -z "$up" ] && {
```

Comment:
I think that when an up or down command is already configured, you should wrap it instead of disabling the hotplug mechanism in this case.